engine/uState.ml

```diff
@@ -826,8 +826,10 @@ let universe_context_inst decl qvars levels names =
   let leftus = List.fold_left (fun acc l -> Level.Set.remove l acc) levels decl.univdecl_instance in
   let () =
     let unboundus = if decl.univdecl_extensible_instance then Level.Set.empty else leftus in
-    if not (QVar.Set.is_empty leftqs && Level.Set.is_empty unboundus)
-    then error_unbound_universes leftqs unboundus names
+    (* allow unbound qualities, cf #19072
+       collapsing ensured that we only get named qualities anyway *)
+    if not (Level.Set.is_empty unboundus)
```

Comment:
This looks like looking for trouble if we want to generalize the use of sort polymorphic quantifications. We would get dangling sorts if we don't pay attention. Shouldn't we fix abstract instead?

Reply:
To change something on the abstract side we would need to introduce a notion of qvar extensible universe declaration
I guess we could but it's more work

Reply:
That'd be more in line with what we do with levels, wouldn't it? If so that may be the better way.

Reply:
changed